.github/workflows/ci.yml

```diff
@@ -61,13 +60,12 @@ jobs:
       - name: Download dependencies
         id: depdownload
         run: |
-          sudo apt-get update
-          sudo apt-get install -y libdevmapper-dev libbtrfs-dev libgpgme-dev
+          hack/build/ci/install_cgo_dependencies.sh
+          hack/build/ci/install_kubebuilder.sh
```

Comment:
why do we need kubebuilder now in these steps?

Reply:
Previously this was done in `test.sh` - separated the installation into an own file.